tests/integration/test_put_along_axis.py

```diff
@@ -54,6 +67,191 @@ def test_ndim(ndim):
         assert np.array_equal(np_a, num_a)
 
 
+def test_full():
+    shape = (3, 4, 5)
+    ndim = len(shape)
+    np_arr = mk_seq_array(np, shape)
+    num_arr = mk_seq_array(num, shape)
+
+    for axis in range(-1, ndim):
+        size = shape[axis]
```

Comment:
can this for loop be moved to pytest's parametrize like:
`@pytest.mark.parametrize("ndim", range(-1, 3))`

Reply:
Fixed.